rush.json

```
@@ -24,7 +24,7 @@
    * Specify one of: "pnpmVersion", "npmVersion", or "yarnVersion".  See the Rush documentation
    * for details about these alternatives.
    */
-  "pnpmVersion": "4.6.0",
+  "pnpmVersion": "4.14.2",
```

```suggestion
  "pnpmVersion": "4.14.1",
```

Can you try using 4.14.1 until we change the version of Node used for build/test?

---

yup, just pshed another commit :)